### PR TITLE
Load gtm only if anonymous_statistics are enabled

### LINKED ISSUE #5577: .opt-out-from-anonymous-statistics still leaks referrer
##### Bug report summary
Creating the file `.opt-out-from-anonymous-statistics ` to opt-out telemetry seems to have no effect.

I have put the file in `/opt/netdata/etc/netdata/.opt-out-from-anonymous-statistics`, `/etc/netdata/.opt-out-from-anonymous-statistics`

I tried to write `all` in the file as specified in a documentation I read but I cannot find anymore.
File is permissioned to netdata user.

I am aware of the documentation here: https://docs.netdata.cloud/docs/anonymous-statistics/ 

Whatever I do, i see the call to `https://www.googletagmanager.com/gtm.js?` 

Additionally, i'd like to report it here for telemetry as well, I also see a call to `https://netdata.cloud/account/sso-agent`, which is not expected as I already run I run my own registry to avoid calls to my-netdata.io

##### OS / Environment
CoreOS Stable

##### Netdata version (ouput of `netdata -V`)
v1.12.1

##### Component Name
netdata 

##### Steps To Reproduce
Install netdata 1.12.1 with release page run.gz on CoreOs Stable, try to disable telemetry as per documentation

##### Expected behavior
I expect netdata to respect my configuration options

### PR DESCRIPTION
**Pull request by cakrit**
##### Summary
Fixes #5577 

##### Component Name
Load gtm.js only if anonymous statistics are enabled.

### CODE REVIEW

File: web/gui/src/dashboard.js/registry.js [hunk 1]
```diff
@@ -84,8 +84,16 @@ NETDATA.registry = {
...
-                    if (data.anonymous_statistics) dataLayer.push({"anonymous_statistics" : "true", "machine_guid" : data.machine_guid});
+                if (!NETDATA.registry.anonymous_statistics_checked) {
+                    NETDATA.registry.anonymous_statistics_checked=true;
+                    if (data.anonymous_statistics) {
+                        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+                                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+                            j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+                            'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+                        })(window,document,'script','dataLayer','GTM-N6CBMJD');
+                        dataLayer.push({"anonymous_statistics" : "true", "machine_guid" : data.machine_guid});
```
Comment (gmosx) [inline web/gui/src/dashboard.js/registry.js]: Are you sure that the script is fully loaded before this line? please note the async=true at line 92.

File: web/gui/src/dashboard.js/registry.js [hunk 2]
```diff
@@ -84,8 +84,16 @@ NETDATA.registry = {
...
                 NETDATA.registry.machine_guid = data.machine_guid;
                 NETDATA.registry.hostname = data.hostname;
-                if (dataLayer) {
-                    if (data.anonymous_statistics) dataLayer.push({"anonymous_statistics" : "true", "machine_guid" : data.machine_guid});
+                if (!NETDATA.registry.anonymous_statistics_checked) {
+                    NETDATA.registry.anonymous_statistics_checked=true;
+                    if (data.anonymous_statistics) {
+                        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+                                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+                            j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
```
Comment (gmosx) [inline web/gui/src/dashboard.js/registry.js]: Note async=true

**Comment by gmosx** [comment]
Maybe you should remove `async` or use `onload`.

File: web/gui/src/dashboard.js/registry.js (hunk 1 above)
Comment (cakrit) [reply web/gui/src/dashboard.js/registry.js]: It's async=false now

File: web/gui/src/dashboard.js/registry.js (hunk 2 above)
Comment (cakrit) [reply web/gui/src/dashboard.js/registry.js]: changed to async=false

**Pull request merged**